src/memalloc.jl

```diff
@@ -23,6 +24,7 @@ function analyze_malloc_files(files)
                 if !isempty(tln) && isdigit(tln[1])
                     s = split(tln)
                     b = parse(Int, s[1])
+					skip_zeros && b == 0 && continue
```

Comment:
Nit but can you use spaces for indent, to match repo style

Reply:
Oh yeah, I forgot to set indent to spaces at my home computer where I made the changes.

I will do it tomorrow!

Thanks for the review.

Reply:
Done!